### ISSUE TITLE
The problem with installing and removing add-ons

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#14619]
### Summary
The problem with installing and removing add-ons

### Step to reproduce
Go to Package Management and try to install and uninstall the add-on.

### Observed behavior

Setup Log:

```
Console running...
Attempting to install package with signature: migx-2.12.0-pl
Package found...now preparing to install.
Grabbing package workspace...
Workspace environment initiated, now installing package...
Could not find legacy class modx.modNamespace after converting to MODX\Revolution\Modx\modNamespace
Could not load class: modNamespace from modx.mysql.modnamespace
Could not load vehicle!
Could not find legacy class modx.modCategory after converting to MODX\Revolution\Modx\modCategory
Could not load class: modCategory from modx.mysql.modcategory
Could not load vehicle!
Could not find legacy class modx.modMenu after converting to MODX\Revolution\Modx\modMenu
Could not load class: modMenu from modx.mysql.modmenu
Could not load vehicle!
Successfully installed package migx-2.12.0-pl
```

When I try to delete, I get an error:
`Error uninstalling package with signature: [[+signature]]`

Uninstall Log:

```
Console running...
Finding package with signature: formit-4.2.3-pl
Package found. Preparing to uninstall.
Could not find legacy class modx.modMenu after converting to MODX\Revolution\Modx\modMenu
Could not load class: modMenu from modx.mysql.modmenu
Problem instantiating object from vehicle
Could not find legacy class modx.modCategory after converting to MODX\Revolution\Modx\modCategory
Could not load class: modCategory from modx.mysql.modcategory
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modSystemSetting after converting to MODX\Revolution\Modx\modSystemSetting
Could not load class: modSystemSetting from modx.mysql.modsystemsetting
Problem instantiating object from vehicle
Could not find legacy class modx.modNamespace after converting to MODX\Revolution\Modx\modNamespace
Could not load class: modNamespace from modx.mysql.modnamespace
Problem instantiating object from vehicle
Error uninstalling package with signature: formit-4.2.3-pl
```

Error log:
```
[2019-06-27 09:18:08] (ERROR in xPDO\Om\xPDOQuery::__construct @ /Volumes/HDD Data/web/refactoring/core/vendor/xpdo/xpdo/src/xPDO/Om/xPDOQuery.php : 116) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class transport.modTransportProvider with MODX\Revolution\Transport\modTransportProvider to take advantage of PSR-4 autoloading.
[2019-06-27 09:18:08] (ERROR in xPDO\xPDO::call @ /Volumes/HDD Data/web/refactoring/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 759) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class transport.modTransportProvider with MODX\Revolution\Transport\modTransportProvider to take advantage of PSR-4 autoloading.
[2019-06-27 09:18:08] (ERROR in xPDO\Om\xPDOObject::load @ /Volumes/HDD Data/web/refactoring/core/vendor/xpdo/xpdo/src/xPDO/Om/xPDOObject.php : 399) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class transport.modTransportProvider with MODX\Revolution\Transport\modTransportProvider to take advantage of PSR-4 autoloading.
[2019-06-27 09:18:11] (ERROR in xPDO\xPDO::getService @ /Volumes/HDD Data/web/refactoring/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 1222) MODX\Revolution\Rest\modRestClient::__construct is deprecated since version 2.3.0. Use the modRest classes instead.
[2019-06-27 09:18:11] (ERROR in xPDO\xPDO::getService @ /Volumes/HDD Data/web/refactoring/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 1222) MODX\Revolution\Rest\modRestClient::__construct is deprecated since version 2.3.0. Use the modRest classes instead.
[2019-06-27 09:18:11] (ERROR in MODX\Revolution\Rest\modRestClient::getConnection @ /Volumes/HDD Data/web/refactoring/core/src/Revolution/Rest/modRestClient.php : 115) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class rest.modRestCurlClient with MODX\Revolution\Rest\modRestCurlClient to take advantage of PSR-4 autoloading.
[2019-06-27 09:18:11] (ERROR in MODX\Revolution\Rest\modRestClient::getConnection @ /Volumes/HDD Data/web/refactoring/core/src/Revolution/Rest/modRestClient.php : 115) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class rest.modRestCurlClient with MODX\Revolution\Rest\modRestCurlClient to take advantage of PSR-4 autoloading.
[2019-06-27 09:18:11] (ERROR in MODX\Revolution\Rest\modRestCurlClient::__construct @ /Volumes/HDD Data/web/refactoring/core/src/Revolution/Rest/modRestCurlClient.php : 31) MODX\Revolution\Rest\modRestClient::__construct is deprecated since version 2.3.0. Use the modRest classes instead.
[2019-06-27 09:18:11] (ERROR in MODX\Revolution\Rest\modRestCurlClient::__construct @ /Volumes/HDD Data/web/refactoring/core/src/Revolution/Rest/modRestCurlClient.php : 31) MODX\Revolution\Rest\modRestClient::__construct is deprecated since version 2.3.0. Use the modRest classes instead.
[2019-06-27 09:18:11] (ERROR in xPDO\xPDO::getService @ /Volumes/HDD Data/web/refactoring/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 1222) MODX\Revolution\Rest\modRestClient::__construct is deprecated since version 2.3.0. Use the modRest classes instead.
[2019-06-27 09:18:11] (ERROR in MODX\Revolution\Rest\modRestClient::getConnection @ /Volumes/HDD Data/web/refactoring/core/src/Revolution/Rest/modRestClient.php : 115) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class rest.modRestCurlClient with MODX\Revolution\Rest\modRestCurlClient to take advantage of PSR-4 autoloading.
[2019-06-27 09:18:12] (ERROR in MODX\Revolution\Rest\modRestCurlClient::__construct @ /Volumes/HDD Data/web/refactoring/core/src/Revolution/Rest/modRestCurlClient.php : 31) MODX\Revolution\Rest\modRestClient::__construct is deprecated since version 2.3.0. Use the modRest classes instead.
```

### Environment
MODX 3 with the latest commit https://github.com/modxcms/revolution/commit/26430024e33db91b599e57a5b9c530b19cd6be31
PHP 7.1.2
MySQL 5.7.23

**Comment by alexsoin** [comment]
I reproduce this error.
[![](https://file.modx.pro/files/e/1/3/e1332cc2a916387722a72e2b51e903b8s.jpg)](https://file.modx.pro/files/e/1/3/e1332cc2a916387722a72e2b51e903b8.png)

**Comment by opengeek** [comment]
Resolved in eb6e44128cba884f6bebda8bee2e4aa1fc8570d0
